frontend/src/scripts/router.ts

```diff
@@ -0,0 +1,9 @@
+import {createRouter, createWebHistory} from 'vue-router';
+import routes from './routes';
+
+const router = createRouter({
```

Comment:
I think we can merge `routes.ts` in `router.ts`.

Reply:
I was pretty sure your would say that 😆 . I was thinking that because there'll be more routes later (maybe a lot), it could be more readable to put them in a dedicated file. However I agree with merging the two files.

Reply:
I don't think i'ts a problem because the only thing `router.ts` is doing is 

```typescript
const router = createRouter({
    history: createWebHistory(),
    routes,
});

export default router;
```
And I suspect that `router.ts` will not change that much.